index.html

```diff
@@ -7276,8 +7276,8 @@ <h2>application/vp</h2>
         </p>
 
         <p>
-This media type can be used for presentations secured using an
-[=enveloping proof=].
+This media type can be used within an [=enveloping proof=] when denoting the
+contents of the envelope.
```

Comment:
Shortening the text and making it easier to read (to address Mike's concern):

```suggestion
This media type can be used in an [=enveloping proof=] to denote the enveloped payload.
```